docs/features.rst

```diff
@@ -40,11 +40,10 @@ features:
 - Client Near Cache Stats
 - Client Runtime Stats
 - Client Operating Systems Stats
-- Hazelcast Cloud Discovery
+- Hazelcast Viridian Discovery
```

Comment:
Do we use this term in Java client docs? @Serdaro Should we update them as well? FYI